py_retro/wave_audio.py

```diff
@@ -61,7 +61,7 @@ def set_audio_sink_batch(core, filenameOrHandle):
     def wrapper(data, frames):
         # We can safely use .writeframesraw() here because the header will be
         # corrected once we call .close()
-        res.writeframesraw(ctypes.string_at(data, frames * 2)[:frames * 2])
+        res.writeframesraw(ctypes.string_at(data, frames * 4)[:frames * 4])
```

Comment:
what are 2 & 4 here? please add a comment or use a named constant

Reply:
also, does this break other cores? should it be configurable?

Reply:
it's always been broken, actually.  naming the constants would be a good idea~
it's basically number of frames * sizeof(int16_t) * stereo